api/src/main/java/org/openmrs/module/bedmanagement/dao/impl/BedManagementDaoImpl.java

```diff
@@ -111,6 +135,16 @@ public Bed getLatestBedByVisit(String visitUuid) {
 		return bed;
 	}
 	
+	@Override
+	public List<Bed> getAssignedBedsByVisit(String visitUuid) {
```

Comment:
The signature of this method does not imply that bed assignments must not be ended.  I would either pass an `includeEnded` parameter, or loosen this restriction

Reply:
I'm glad you made me changed this. Refactored and found a bug in the `includeEnded` logic elsewhere.